package.json

```diff
@@ -119,7 +119,7 @@
     "@elastic/ecs": "^8.11.1",
     "@elastic/elasticsearch": "^8.15.0",
     "@elastic/ems-client": "8.5.3",
-    "@elastic/eui": "97.0.0-backport.2",
```

Comment:
Can we also update https://github.com/elastic/kibana/blob/8.16/src/dev/license_checker/config.ts#L90

Reply:
absolutely! thank you for letting me know!